src/main/java/uk/gov/justice/digital/delius/service/AttendanceService.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.justice.digital.delius.service;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.justice.digital.delius.data.api.Attendance;
+import uk.gov.justice.digital.delius.data.api.Attendance.ContactTypeDetail;
+import uk.gov.justice.digital.delius.jpa.standard.entity.Contact;
+import uk.gov.justice.digital.delius.jpa.standard.repository.ContactRepository;
+
+@Slf4j
+@Service
+public class AttendanceService {
+
+    private final ContactRepository contactRepository;
+
+    @Autowired
+    public AttendanceService(final ContactRepository contactRepository) {
+        this.contactRepository = contactRepository;
+    }
+
+    public Optional<List<Contact>> getContactsForEvent(final Long eventId, final LocalDate localDate) {
+        final List<Contact> contacts = contactRepository.findByEventIdEnforcement(eventId, localDate);
+        return contacts.isEmpty() ? Optional.empty() : Optional.of(contacts);
```

Comment:
I wonder if just returning a list is better? So if they have no attendance should that return a 404 or empty list?

 I suspect it should be 404 if the offender (see above comment) or event/conviction not found but 200 if no attendances found

Reply:
OK - will change.

Reply:
Updated - empty list now returns 200